packages/components/src/VirtualizedList/CellDatetime/CellDatetime.test.js

```diff
@@ -64,4 +64,31 @@ describe('CellDatetime', () => {
 		expect(computedStrOffset).toEqual(expectedStrDate);
 		expect(format).toHaveBeenCalledWith(cellDataWithOffset, columnData.pattern);
 	});
+
+	it('should test CellDatetime render with tooltip', () => {
+		// when
+		const columnData = {
+			mode: 'ago',
+			fullDateInTooltip: true,
```

Comment:
I would go to always have the full date in tooltip.
Why make it an option ?

Reply:
It makes sense, I removed the option.